02-FlexBox/assignment/package.json

```diff
@@ -17,5 +17,8 @@
         "gulp": "4.0.2",
         "gulp-sass": "^5.1.0",
         "sass": "^1.50.1"
+    },
+    "dependencies": {
```

Comment:
This might be an accident but be careful adding in extra dependencies that aren't needed.